provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -414,10 +414,12 @@ def _get_node_clients(self, machine_id: str) -> Iterator[M0ClientDesc]:
         """
         for client in self.provider.get_motr_clients():
             name = str(client.get('name'))
-            if self.utils.is_component(machine_id, name):
-                yield M0ClientDesc(
-                    name=Text(name),
-                    instances=int(str(client.get('num_instances'))))
+            instances = int(str(client.get('num_instances')))
+            if instances:
+                if self.utils.is_service(machine_id, name):
+                    yield M0ClientDesc(
+                        name=Text(name),
+                        instances=int(str(client.get('num_instances'))))
```

Comment:
```suggestion
                        instances=no_instances)
 ```
use the above variable.

Reply:
I will add in the PR. Thanks!